### PR TITLE
Do not steal the focus when showing the channels

### PR DESCRIPTION
**Pull request by akshita31**
Fixes: https://github.com/OmniSharp/omnisharp-vscode/issues/2756

### CODE REVIEW

File: test/unitTests/logging/OmnisharpChannelObserver.test.ts [hunk 1]
```diff
@@ -11,14 +11,19 @@ suite("OmnisharpChannelObserver", () => {
...
     let observer: OmnisharpChannelObserver;
 
     setup(() => {
         hasShown = false;
         hasCleared = false;
+        preserveFocus = false;
         observer = new OmnisharpChannelObserver({
             ...getNullChannel(),
-            show: () => { hasShown = true; },
+            show: (preserve) => {
```
Comment (rchande) [inline test/unitTests/logging/OmnisharpChannelObserver.test.ts]: Do we ever call this with false?

Comment (akshita31) [reply test/unitTests/logging/OmnisharpChannelObserver.test.ts]: Is there a case where we want to steal the focus

Comment (akshita31) [reply test/unitTests/logging/OmnisharpChannelObserver.test.ts]: As of now I couldnt think of any particular case hence right now we are using true everywhere

Comment (rchande) [reply test/unitTests/logging/OmnisharpChannelObserver.test.ts]: Why not make it even easier and remove the paramter/always pass true in this method?

**Comment by filipw** [comment]
this is great - thanks!

**Pull request merged**